lib/plug/cowboy/drainer.ex

```diff
@@ -38,6 +62,7 @@ defmodule Plug.Cowboy.Drainer do
 
   @doc false
   def start_link(opts) do
+    Keyword.fetch!(opts, :refs)
```

Comment:
Let's validate if it is `:all` or a list, otherwise it will fail only on shutdown, which will be hard to figure it out. :)